.github/workflows/matrix-and-codecov-on-merge-to-main.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - prereleased
+      - published
+  workflow_dispatch: null
+
+jobs:
+  coverage:
+    uses: Billingegroup/release-scripts/.github/workflows/_matrix-and-codecov-on-merge-to-main.yml@v0
+    with:
+      project: diffpy.structure
+      c_extension: false
```

Comment:
Please check. I think this maybe should be yes

Reply:
done